src/tkn.ts

```diff
@@ -76,7 +76,7 @@ export enum ContextType {
   CONDITIONSNODE = 'conditionsnode',
   CONDITIONS = 'conditions',
   PIPELINERUNNODE = 'pipelinerunnode',
-  CONDITIONRUN = 'conditions',
+  CONDITIONRUN = 'tr',
```

Comment:
Maybe `tr` is not the best name, I think `conditionaltaskrun` or `conditionrun` is more descriptive

Reply:
I am using `tr` to open yaml in a text editor.

Reply:
Yes, I mean `CONDITIONRUN` doesn't not corresponding to `tr`